build/site/specifications/2.1/specs/C2PA_Specification.html

```diff
@@ -3497,11 +3497,11 @@ <h4 id="_signature_algorithms"><a class="anchor" href="#_signature_algorithms"><
 <div class="ulist">
 <ul>
 <li>
-<p>ECDSA requires elliptic curve keys on the P-256, P-384, or P-521 elliptic curves.</p>
+<p>ECDSA requires elliptic curve keys on the P-256, K-256, P-384, or P-521 elliptic curves.</p>
```

Comment:
I need to do a bit of research to refresh my memory, but the typical concern with K-256 is that there are a few scenarios where timing attacks could leak information. As used in crypto-currency implementations, the timing attacks are impractical. Generalized use in C2PA might??? be problematic in some scenarios. Leonard, let me know if you want me to dig up the details or run this by cryptographers I work with.